schemas/EventId.json

```diff
@@ -3,5 +3,5 @@
   "title": "EventId",
   "description": "An identifier for a subscription event.",
   "type": "string",
-  "pattern": "^(\\*|[a-z]+\\.(\\*|[a-z]+))$"
+  "pattern": "^(\\*|[a-z]+)$"
```

Comment:
When notifications were changed from HTTP to websocket, the event type labels were removed.
When adding them back I decided to use just `"transfer"` since no other "transfer" events existed besides `transfer.update`, and it fits better with the other event names.